doc/api/console.markdown

```diff
@@ -109,6 +109,43 @@ console.assert(false, 'Whoops %s', 'didn\'t work');
   // AssertionError: Whoops didn't work
 ```
 
+It is important to note that the `console.assert()` method in Node.js is
+implemented differently than the `console.assert()` method available in
+browsers. Specifically, in browsers, calling `console.assert()` with a falsy
+assertion will cause the `message` to be printed to the console without
+interrupting execution of subsequent code. In Node.js, however, a falsy
+assertion will cause an `AssertionError` to be thrown.
+
+Functionality approximating that implemented by browsers can be implemented
+by extending Node.js' `console` and overriding the `console.assert()` method.
+
+In the following example, a simple module is created that extends and overrides
+the default behavior of `console` in Node.js.
+
+```js
+'use strict';
+
+const myConsole = Object.setPrototypeOf({
```

Comment:
actually, the example is incomplete... just noticed... it should be:

```js
const myConsole = Object.setPrototypeOf({
   assert(assertion, message, ...args) {
     try {
      console.assert(assertion, message, ...args);
     } catch (err) {
       console.error(err.stack);
    }
   }
 }, console);
```
Essentially, this creates a simple extension of the existing console object without monkey-patching it.